features/0509-action-menu/README.md

```diff
@@ -0,0 +1,211 @@
+  "@id": "5678876542347",
+  "~thread": {
+    "thid": "5678876542344"
+  },
+  "title": "Attendance Verified",
+  "description": "",
+  "options": [
+    {
+      "name": "submit-invitation",
+      "title": "Submit an invitation",
+      "description": "Send an invitation for IIWBook to share with another participant"
+    },
+    {
+      "name": "search-introductions",
+      "title": "Search introductions",
+      "form": {
+        "description": "Enter a participant name below to perform a search.",
+        "params": [
+          {
+            "name": "query",
+            "title": "Participant name",
+            "default": "",
+            "description": "",
+            "required": true,
+            "type": "text"
+          }
+        ],
+        "submit-label": "Search"
+      }
+    }
+  ]
+}
+```
+
+When the form is submitted, a _perform_ message is generated containing values entered in the form. The `form` block may have an empty or missing `params` property in which case it acts as a simple confirmation dialog.
+
+Each entry in the `params` list must define a `name` and `title`. The `description` is optional (should be displayed as help text below the field) and the `type` defaults to ‘text’ if not provided (only the ‘text’ type is supported at this time). Parameters should default to `required` true, if not specified. Parameters may also define a `default` value (used when rendering or clearing the form).
+
+
+#### menu-request
+
+In addition to menus being pushed by the `responder`, the root menu can be re-requested at any time by the `requestor` sending a _menu-request_.
+
+```jsonc
+{
+  "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/action-menu/1.0/menu-request",
+  "@id": "5678876542345"
+}
+```
+
+#### perform
+
+When the `requestor` user actions a menu option, a _perform_ message is generated. It should be attached to the same thread as the _menu_. The active _menu_ should close when an option is selected.
+
+The response to a _perform_ message can be any type of agent message, including another menu message, a presentation request, an introduction proposal, a credential offer, an acknowledgement, or a problem report. Whatever the message type, it should normally reference the same message thread as the perform message.
+
+```jsonc
+{
+  "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/action-menu/1.0/perform",
+  "@id": "5678876542346",
+  "~thread": {
+    "thid": "5678876542344"
+  },
+  "name": "obtain-email-cred",
+  "params": {}
+}
+```
+
+##### Description of attributes
+
+* `name` -- the _menu_ option being requested. This is taken from the `name` attribute of the `options` array elements in the _menu_  
+* `params` -- optional dictionary containing any input parameters requested in a _menu_ option `form` section.  The dictionary key values are taken from the `name` attributes of `params` array elements in the _menu_ option `form`.
+
+## Drawbacks
+
+Why should we *not* do this?
```

Comment:
Same comment as above with removing the template instructions and putting in a blanks or "None" or "N/A" or something like that.

Reply:
Done